src/common/PeopleSection/styled.js

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+
+export const Title = styled.h1`
+    font-size: 36px;
+    line-height: 120%;
+    margin: 0 0 32px;
+
+    @media(max-width: ${({theme}) => theme.breakpoints.xsmall}px) {
+        font-size: 20px;
+        margin-bottom: 12px;
+    }
+`;
+
+export const PeopleList = styled.ul`
+    display: grid;
+    grid-gap: 24px;
+    grid-template-columns: repeat(6, 1fr);
+    list-style-type: none;
+    padding: 0;
+
+    @media(max-width: ${({theme}) => theme.breakpoints.large}px) {
```

Comment:
i think it would better work on this conditions:
   @media(max-width: ${({ theme }) => theme.breakpoints.large}px) {
        grid-template-columns: repeat(4, 1fr);
    }

    @media(max-width: ${({ theme }) => theme.breakpoints.medium}px) {
        grid-template-columns: repeat(3, 1fr);
    }

    @media(max-width: ${({ theme }) => theme.breakpoints.xsmall}px) {
        grid-template-columns: repeat(2, 1fr);
    }